### DISCUSSION TITLE
Welcome to Terminus Discussions!

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#2732] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share ideas and engage with other community members.
  * Ask questions you’re wondering about that don't necessarily fit under Issues.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alxz99** [comment]
Hello Eugeny!

I wanted to say thank you for this amazing project. I just started to use it yesterday. 

I would like you to add a third division for the section connections, because i work with diferents customers and i like to have everything in it respective position. 

Right now Termius only can divide connetions in two sections: grups and connections. 

Why not a third (or maybe a 4th haha) Like Group, Subgroup, Connections. 
![printscreen](https://user-images.githubusercontent.com/69919303/90673450-fff9b680-e214-11ea-909d-18337ad59e67.JPG)

**Comment by noehmeier** [comment]
Hi, i am not that ssh professional, but my company wants us to connect to a server over a jump server. So i have to configure port forwarding, with a config like this:

Hostname my.company.server.com
LocalForward 33210 127.0.0.1:22
IdentityFile ~/.ssh/id_rsa
User my.user.name

I tried to set this config over the Terminus GUI at the SSH section where i added a connection like this:

![grafik](https://user-images.githubusercontent.com/7363107/97608562-cca48680-1a12-11eb-844a-1b9477a942e7.png)

As default, he reads correctly my ssh-key from ~/.ssh/id_rsa.
But where i have to set the LocalForward-part?
In an issue (https://github.com/Eugeny/terminus/issues/2077) i found the phrase "The local/remote portforwarding options in terminus are very very nice" so there must be options. Where can i find them or what i have to do? I would be happy if someone can help me out :-)

**Reply by Eugeny** [reply]
You don't have to manually set up port forwarding to use jump hosts in Terminus - just set up your jump host as if it were a normal host, and then set it as jump host for the actual connection you want to use it with.

**Reply by amajauskas** [reply]
I have it set up as such,

Set up the jump host as a normal host and then set up port forwarding for that you are trying to reach.

Is this what you mean?

This is how I have it set up as of now, and it works per se, but it's sort of counterintuitive as it's all in one pile. I mean connections to hosts and connections used for port forwarding. I was wondering if some sort of separation could be implemented? Maybe persistent (hostname -> connection name) naming option? Would definitely help spot the difference.

**Reply by Eugeny** [reply]
Of course you can visually label your jump host as `[jump host]`, or put then in a separate group, but I think I'm misunderstanding your point - could you elaborate? 

The phrase in your original post was referring to normal port forwarding, not jump hosts: 
![image](https://user-images.githubusercontent.com/161476/118522152-0aad9180-b73c-11eb-8ee9-2def18eb558b.png)

**Reply by amajauskas** [reply]
Actually seems like this is already implemented with the "title-control" plugin - thanks to James Brumond. Allows you to keep persistent naming of the tab - displayed in the tab as in the entry.

But I guess I was also wondering if a port forwarding implementation could be added that is similar to "Termius" where you dont need a whole new connection tab, just something you enable/disable. Seems like the functionality would be the same, its just a change on the front end, how its handled.

**Reply by Eugeny** [reply]
I get it now. Feel free to post a separate issue for this - somebody might come around to implement it

**Comment by KeKouShi** [comment]
hi,Whether the software can add Chinese features?
